docs/release-notes/charmcraft-3.5.rst

```diff
@@ -0,0 +1,161 @@
+Charmcraft 3.5 release notes
+============================
+
+**UNRELEASED**. Expected release date: 7 April 2025
+
+Learn about the new features, changes, and fixes introduced in Charmcraft 3.5.
+
+
+Requirements and compatibility
+------------------------------
+
+For development and testing, Charmcraft requires a host with a minimum of 4GB RAM
+running a Linux distribution compatible with systemd.
+
+All versions of Charmcraft require the following software:
+
+- systemd
+- `snapd`_
+- Either `LXD`_ or `Multipass`_
+
+We recommend you install the `Charmcraft snap <https://snapcraft.io/charmcraft>`_. It
+comes comes bundled with all its dependencies.
+
+Non-snap installations of Charmcraft have the following dependencies:
+
+- Python 3.10 or higher
+- libgit2 1.7
+- `skopeo`_
+- `Spread`_
+
+
+What's new
+----------
+
+Charmcraft 3.5 brings the following features, integrations, and improvements.
+
+
+Updated ``init`` templates
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The templates generated by ``charmcraft init`` now use the new
+:doc:`/reference/platforms` format.
+
+
+JSON Schema
+~~~~~~~~~~~
+
+The `JSON Schema Store <https://schemastore.org>`_ now includes a schema for the :ref:`charmcraft-yaml-file`. If your text editor supports getting JSON schemas from
```

Comment:
Are we also updating the schema we ship in the package? IDEs support loading schemas as a local file.